en/app.ftl

```diff
@@ -821,9 +821,9 @@ multi-part-onboarding-reply-headline = Reply to your emails
 
 multi-part-onboarding-premium-welcome-description-2 = No more five-mask limit: you can now generate as many custom or random masks as you need. On desktop, you can use the { -brand-name-relay } add-on to create them on the go.
 # Variables:
-#   $mozmail (string): domain used by Relay masks (.mozmail.com)
+#   $mozmail (string): domain used by Relay masks (mozmail.com)
 multi-part-onboarding-premium-get-domain-description-3 = With a custom subdomain, you can create masks without having to generate them beforehand.
```

Comment:
The change needs to have a new string ID even though it's almost identical to the old version. Without the new ID, the change won't trigger the locales that had localized the string. You can name it `…-description-2b' or something like this. And the old string should be removed so it doesn't get localized if no longer used.

Reply:
Oops, you're right. Just added a new line.